setup.py

```diff
@@ -1,6 +1,6 @@
 import pathlib
 from setuptools import setup, find_packages
-import sys
+
```

Comment:
```suggestion
```

Reply:
I prefer to keep two empty lines between imports and code like in other python files.

Reply:
ok